### ISSUE TITLE
Export IRC renewable energy datasets into Google sheet

### ISSUE THREAD

**Issue by monika-aashe** [AASHE/hub#292]
In preparation for round 2 of IRC to Hub exports, please send a link to a Google Sheet file of the following renewable energy datasets to Monika for formatting before importing into the Hub:

- Solar photovoltaic installations database (all installations, 585 solar photovoltaic installations on 333 campuses) - http://www.aashe.org/resources/campus-solar-photovoltaic-installations/all/
- Campus fuel cell installations (all 12 installations) - http://www.aashe.org/resources/campus-installations-stationary-fuel-cells
- Wind turbine installations (all installations - about 81) - http://www.aashe.org/resources/wind-power-campus-1

Considerations:

- Please complete files by Monday, April 25
- Please include all data fields collected through forms or included on the dataset
- Please send each list as a separate tab on the same Google sheet file).
- Please ensure that nothing is missing from the lists (see counts above).
- Please try to remove any special character issues.
- If possible, include the date and time the IRC resource was submitted and/or approved.

**Comment by charris2015** [comment]
Hey Monika,

Here's a [link](https://drive.google.com/a/aashe.org/folderview?id=0B-OBXNpaVv-OMDdQYUpncVZnblU&usp=sharing) to the file containing the renewable energy datasets. I tried to add the org names to the exports but many of them didn't have information that I could use to match to our organizations table. I was able to match account names to the solarpv export but the export that's matched with the org names (solarpv_solarinstallation.csv) has about 250 entries less than the original.
